dotnetv3/cross-service/PhotoAssetManager/README.md

```diff
@@ -0,0 +1,106 @@
+  see [Setting up your AWS SDK for .NET environment](https://docs.aws.amazon.com/sdk-for-net/v3/developer-guide/net-dg-setup.html).
+
+## Create the resources
+
+Follow the instructions in the
+[README for the Photo Asset Manager application CDK](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/applications/photo-asset-manager/cdk/README.md).
+to use the AWS Cloud Development Kit (AWS CDK) or AWS Command Line Interface
+(AWS CLI) to create and manage the resources used in this example. You must be running Docker in order to complete the steps for this CDK.
+
+## Build the code
+
+The Visual Studio solution **PhotoAssetManager.sln** includes several projects with different purposes for this example.
+
+- **PamApi** - A Lambda Serverless Api project that includes an ApiController for the serverless endpoints, a Swagger UI for development environments, and the separate Download and DetectLabels functions. 
+  - This is the project that will be packaged and deployed when using the `dotnet` language option when setting up the CDK stack.
+- **PamApiAnnotations** - A [Lambda Annotations](https://github.com/aws/aws-lambda-dotnet/blob/master/Libraries/src/Amazon.Lambda.Annotations/README.md) project that includes a version of the serverless endpoints that use Lambda Annotations for .NET.
+  - This is the project that will be packaged and deployed when using the `dotnetla` language option when setting up the CDK stack.
+- **PamServices** - A C# library project that includes shared classes and custom services and business logic.
+  - The services in this project are injected into the Api projects' dependency injection setup in the Startup.cs files.
+- **PamServices.Test** - An XUnit test project with unit tests for the PamServices project classes.
+
+### Local development
+The PamApi project includes a Swagger integration and setup, with additional test endpoints that can be used for investigating the various application functionality.
+You can access this UI by running the `PamApi` project from Visual Studio in debug mode. 
+
+![pam_api_swagger.png](pam_api_swagger.png)
+
+For local development, you will also need to set the following environment variables that are automatically set for you during CDK deployment.
+
+* LABELS_TABLE_NAME
+* STORAGE_BUCKET_NAME
+* WORKING_BUCKET_NAME
+* NOTIFICATION_TOPIC
+
+For local debugging of the `PamApiAnnotations` project and the `Download` and 
+`DetectLabels` functions, you can use the [Mock Lambda Test Tool](https://github.com/aws/aws-lambda-dotnet/blob/master/Tools/LambdaTestTool/README.md).
+To run the tool for debugging, select the tool from the `Debug` menu in Visual Studio.
+
+![pam_mock_lambda.png](pam_mock_lambda.png)
+
+### Application notes
+* The maximum pixel size for analysis is 10000x10000. Larger images will not be analyzed.
+* Some email clients do not support the long length download urls. You may need to remove any spaces from the url before downloading.
```

Comment:
* Some email clients do not support the long length download URLs. You might need to remove spaces from the URL before downloading.